### PR TITLE
OPS-6102 Update the version.json for Aerospike Connect for Presto - Trino 4.1.0

### PR DESCRIPTION
**Pull request by mcoberly2**

### CODE REVIEW

**Comment by mcoberly2** [comment]
@reugn 
Please add @mcoberly2  @pvinh-spike @ishaanb to the People for this trino-aerospike.docker repositoy.
We do not have any rights to even request reviewers or be requested as reviewers.

Additionally, @reugn please review & @ishaanb please review when you gain rights.

File: version.json [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+    "379": "4.1.0-370",
+    "380": "4.1.0-370",
+    "381": "4.1.0-370",
+    "382": "4.1.0-370",
+    "383": "4.1.0-370",
+    "384": "4.1.0-370",
+    "385": "4.1.0-370",
+    "391": "4.1.0-391",
+    "392": "4.1.0-391",
+    "393": "4.1.0-391"
```
Comment (reugn) [inline version.json]: Add
```json
    "394": "4.1.0-391",
    "395": "4.1.0-391"
```

**Review by reugn — CHANGES_REQUESTED** [review]

File: version.json (hunk 1 above)
Comment (mcoberly2) [reply version.json]: updated

**Review by pvinh-spike — APPROVED** [review]
LGTM

**Pull request merged**